webapp/components/admin_console/admin_team_members_dropdown.jsx

```diff
@@ -92,10 +92,7 @@ export default class AdminTeamMembersDropdown extends React.Component {
 
     handleMakeActive(e) {
         e.preventDefault();
-        Client.updateActive(this.props.user.id, true,
-            () => {
-                AsyncClient.getUser(this.props.user.id);
-            },
+        updateActive(this.props.user.id, true, false,
```

Comment:
Minor, but can you use `null` instead of `false` for the argument for the empty success function? It's a little more telling that the argument is meant to empty and for consistency's sake since we do it that way everywhere else